javascript-functions/main.js

```diff
@@ -0,0 +1,44 @@
+function convertMinutesToSeconds(minutes) {
+  const seconds = minutes * 60;
+  console.log(
+    'The number of seconds in ' +
+      minutes +
+      ' minutes is ' +
+      seconds +
+      ' seconds.'
+  );
+}
+convertMinutesToSeconds(5);
```

Comment:
instead of running the `console.log` inside the function, use a `return` statement to return seconds out of the function.

Then on line 11, store the result of the function call in a variable like this:
`const convertMinutesToSecondsResult = convertMinutesToSeconds(5);`

Then `console.log('convertMinutesToSecondsResult', convertMinutesToSecondsResult)'`